### PR TITLE
Adds ProxyCrawl module

### PR DESCRIPTION
**Pull request by proxycrawl**
A couple of our clients asked for Infinity Dashboard integration so here we go 👍

### CODE REVIEW

File: community/proxycrawl_stats.infinitymodule/package.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/proxycrawl/proxycrawl-infinity.git"
+  },
+  "user_options": [
+    {
+      "title": "ProxyCrawl normal token",
+      "name": "normalToken",
+      "type": "string",
+      "tooltip": "Your ProcyCrawl normal token",
```
Comment (emilylaguna) [inline community/proxycrawl_stats.infinitymodule/package.json]: Can you please fix the typo here?

File: community/proxycrawl_stats.infinitymodule/package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      "required": true,
+      "options": {
+        "secure": true
+      }
+    },
+    {
+      "title": "ProxyCrawl javascript token",
+      "name": "javascriptToken",
+      "type": "string",
+      "tooltip": "Your ProcyCrawl javascript token",
```
Comment (emilylaguna) [inline community/proxycrawl_stats.infinitymodule/package.json]: Can you please fix the typo here?

File: community/proxycrawl_stats.infinitymodule/package.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      "type": "string",
+      "tooltip": "Your ProcyCrawl normal token",
+      "default": "",
+      "required": true,
+      "options": {
+        "secure": true
+      }
+    },
+    {
+      "title": "ProxyCrawl javascript token",
```
Comment (emilylaguna) [inline community/proxycrawl_stats.infinitymodule/package.json]: This option gets clipped when loaded in the app, perhaps change the titles to just 'Normal Token' and 'Javascript Token'? The tooltip can provide the the user with the rest of the information about this preference.

screen shot 2018-06-13 at 9 52 00 am

**Comment by proxycrawl** [comment]
@emilylaguna thanks for the review. We've proceeded to do the required changes.

**Comment by emilylaguna** [comment]
Thank you @proxycrawl I've added your module in, we'll be updating the website with the latest community modules later today: http://infinitydashboard.com/

**Pull request merged**